packages/expo-modules-autolinking/.eslintrc.js

```diff
@@ -1,2 +1,8 @@
-// @generated by expo-module-scripts
-module.exports = require('expo-module-scripts/eslintrc.base.js');
+const config = require('expo-module-scripts/eslintrc.base.js');
+
+config.rules = {
+  ...config.rules,
+  'sort-imports': ['error', { 'ignoreDeclarationSort': true }],
```

Comment:
Is there any reason to override this? If so, maybe add a comment?

Reply:
it seems expo-module-scripts does not turn on the sort-imports. for large imports it's good to have. i'm removing this anyway.